lms/services/lti_h.py

```diff
@@ -22,41 +23,84 @@ def __init__(self, _context, request):
         self._lti_user = request.lti_user
         self._h_user = request.lti_user.h_user
 
+        self._authority = request.registry.settings["h_authority"]
         self._ai_getter = request.find_service(name="ai_getter")
         self._h_api = request.find_service(name="h_api")
         self._group_info_service = request.find_service(name="group_info")
 
-    def sync(self, groups):
+    def sync(self, h_groups):
         """
         Sync standard data to H for an LTI launch with the provided groups.
 
         This will upsert the provided list of groups, the current user and
         make that user a member of each group.
 
-        :param groups: A list of models.HGroup objects.
+        :param h_groups: A list of models.HGroup objects.
         :raises HTTPInternalServerError: If we cannot sync to H for any reason
         """
 
         if not self._ai_getter.provisioning_enabled():
-            return None
+            return
 
         try:
-            return self._sync_to_h(groups)
+            self._sync_to_h(h_groups)
```

Comment:
We don't actually return anything and this allows is to split up for the bulk/serial flag and then do the GroupInfo stuff once here.